Fix HDMI

Robin5577 opened the pull request:
Hello, it has been verified, hdmi is available

Robin5577 committed:
Update config.plist

fix hdmi

Yanshuaibing commented:
![2031624083066_ pic](https://user-images.githubusercontent.com/7920189/122632903-3ab7ce00-d108-11eb-82bc-2734fb9e29a0.jpg)
![2041624083067_ pic](https://user-images.githubusercontent.com/7920189/122632907-3e4b5500-d108-11eb-9311-63f1132536cb.jpg)

Yanshuaibing commented:
![2051624083140_ pic_hd](https://user-images.githubusercontent.com/7920189/122632934-60dd6e00-d108-11eb-9d39-31029e3db324.jpg)

AniKulkarn committed:
Failed Attempt #1 to solve HDMI

Failed attempt to solve HDMI. DO NOT MERGE TO MAIN BRANCH. Refer this Slack thread for more details - https://hackintoshthinkpade14.slack.com/archives/C024KBC6N4W/p1624026706007700

Co-Authored-By: Yanshuaibing <7920189+yanshuaibing@users.noreply.github.com>
Co-Authored-By: fdbsilva <83889715+fdbsilva@users.noreply.github.com>

AniKulkarn reviewed (commented):
I have tried booting from this. It did not solve the HDMI problem. I have also added my own commit on top of this by referring to [this Reddit thread](https://www.reddit.com/r/hackintosh/comments/m4us0n/no_hdmi_output_comet_lake_i510210u_and_intel_uhd/grrbq5d?utm_source=share&utm_medium=web2x&context=3). Comments are most welcomed in [this Slack thread](https://hackintoshthinkpade14.slack.com/archives/C024KBC6N4W/p1624026706007700) or in the current GitHub branch.

Yanshuaibing commented:
![2101624168148_ pic](https://user-images.githubusercontent.com/7920189/122663599-5e454c00-d1ce-11eb-8b68-7c11db4a6fff.jpg)
![2091624168147_ pic](https://user-images.githubusercontent.com/7920189/122663596-5be2f200-d1ce-11eb-95fb-fa9e8c311c51.jpg)
![2081624168146_ pic](https://user-images.githubusercontent.com/7920189/122663593-584f6b00-d1ce-11eb-9bf7-438e65f2b6e8.jpg)

hi~, I used the latest master code, only modified the Device part, HDMI can be driven normally.

Zer0xPoint commented:
your device looks differ to config file, your device shows that you are using MacBook Air 2020, but in config file MacBook Pro ,can you provide your config file?

AniKulkarn committed:
Trying out latest HDMI patch

Trying out possible HDMI fix.

Co-Authored-By: Yanshuaibing <7920189+yanshuaibing@users.noreply.github.com>

AniKulkarn commented:
> your device looks differ to config file, your device shows that you are using MacBook Air 2020, but in config file MacBook Pro ,can you provide your config file?

We talked about this on [this Slack thread](https://hackintoshthinkpade14.slack.com/archives/C024KBC6N4W/p1624268958000700). He said it does not matter which SMBIOS you use. I'm trying this commit right now on my laptop. I have a MacBook Pro SMBIOS. Let's see if it works.

huasor commented:
![image](https://user-images.githubusercontent.com/34728738/141606226-3aeec93c-b469-4942-934e-481a9541e703.png)
this config is ok. can you try booting in your PC? @AniKulkarn 
My PC, the CPU is i7-10510U, the dGPU is AMD Radeon RX 640

huasor commented:
refer to this link: https://www.reddit.com/r/hackintosh/comments/hu46xm/fix_hdmi_port_and_possibly_others_not_working_on/
